### PR TITLE
Reveal anchor focus fix

### PR DESCRIPTION
**Pull request by Owlbertz**
Focus active anchor rather than last one in collection when closing a Reveal modal.

Addresses #10604.

### CODE REVIEW

**Review by ncoden — APPROVED** [review]
LGTM

**Comment by ncoden** [comment]
> IamManchanda self-requested a review on 12 Oct

@IamManchanda Would you like to review this PR or do I merge it ?

**Comment by IamManchanda** [comment]
Taking this up @ncoden

File: js/foundation.reveal.js [hunk 1]
```diff
@@ -228,6 +228,9 @@ class Reveal extends Plugin {
       }
     }
 
+    // Remember anchor that opened it to set focus back later, have general anchors as fallback
+    this.$activeAnchor = $(document.activeElement).is(this.$anchor) ? $(document.activeElement) : this.$anchor;
```
Comment (IamManchanda) [inline js/foundation.reveal.js]: 👍

**Review by IamManchanda — APPROVED** [review]
@ncoden @Owlbertz I haven't entered into testing as of yet

But visual testing is working cool and logic seems cool too!

So approved from my end... Thanks!

**Comment by IamManchanda** [comment]
I think we should merge it @ncoden

**Comment by ncoden** [comment]
@IamManchanda This PR do not install or update any package. Why updating the yarn lock file there ?

**Pull request merged**